specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/datamigration.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "title": "Azure Data Migration Service Resource Provider",
```

Comment:
Can you update the terminology to 'Database Migration Service' instead of 'Data Migration Service'. The GA swagger already has this.

Reply:
Done.